src/peergos/server/cli/CLI.java

```diff
@@ -360,7 +365,37 @@ public String buildPrompt() {
                 .append(" > ").toAnsi();
     }
 
-    private List<String> lsForRemoteFilesCompleter(String pathArgument) {
+    private List<String> localFilesLsFiles(String pathArgument) {
+        try {
+            Path path = resolveToPath(pathArgument).toAbsolutePath();
+            if (path.toFile().isFile())
+                return Arrays.asList(path.toString());
+            if (path.toFile().isDirectory())
+                return Files.list(path)
+                        .map(Path::toString)
+                        .collect(Collectors.toList());
+
+            if (!path.getParent().toFile().isDirectory())
```

Comment:
when is something both not a file and not a directory?

Reply:
Yes - if it  a partial path (ie. user hasn't finished typing it)